frontend/packages/console-app/src/queries.ts

```diff
@@ -28,7 +28,7 @@ export const PressureQueries = {
       model: PodModel,
       fieldSelector: `spec.nodeName=${node}`,
       metric: 'pod',
-      query: `(sort_desc(topk(25,sum by(pod, namespace) (container_memory_working_set_bytes{node="${node}"}))))`,
+      query: `(sort_desc(topk(25,sum by(pod, namespace) (container_memory_working_set_bytes{node="${node}", container=""}))))`,
```

Comment:
I believe this is correct. Some places we use `sum(container_memory_working_set_bytes{container=""})` and some places `sum(container_memory_working_set_bytes{container!=""})`. I assume `container!=""` is the total for the pod so both are equivalent? @openshift/openshift-team-monitoring can you confirm?

I don't see any other places where we leave `container` off of the query.

Reply:
> I believe this is correct. Some places we use `sum(container_memory_working_set_bytes{container=""})` and some places `sum(container_memory_working_set_bytes{container!=""})`. I assume `container!=""` is the total for the pod so both are equivalent? @openshift/openshift-team-monitoring can you confirm?

correct, the value without the `container` label should be equal to the sum of all values where the `container` isn't empty.